drivers/gpio/gpio_stm32.c

```diff
@@ -221,6 +221,49 @@ int gpio_stm32_configure(uint32_t *base_addr, int pin, int conf, int altf)
 	return 0;
 }
 
+/**
+ * @brief GPIO port clock handling
+ */
+int gpio_stm32_clock_request(const struct device *dev, bool on)
+{
+	const struct gpio_stm32_config *cfg = dev->config;
+	int ret = 0;
+
+	if (dev == NULL) {
+		ret = -ENODEV;
+		goto exit;
+	}
+
+	/* enable clock for subsystem */
+	const struct device *clk = DEVICE_DT_GET(STM32_CLOCK_CONTROL_NODE);
+
+	if (on) {
+		ret = clock_control_on(clk,
+					(clock_control_subsys_t *)&cfg->pclken);
+
+	} else {
+		ret = clock_control_off(clk,
+					(clock_control_subsys_t *)&cfg->pclken);
+	}
+
+#if defined(PWR_CR2_IOSV) && DT_NODE_HAS_STATUS(DT_NODELABEL(gpiog), okay)
+	z_stm32_hsem_lock(CFG_HW_RCC_SEMID, HSEM_LOCK_DEFAULT_RETRY);
+	if (cfg->port == STM32_PORTG) {
+		/* Port G[15:2] requires external power supply */
+		/* Cf: L4XX RM, §5.1 Power supplies */
```

Comment:
could the comment be like in 5105bcf521a9cd2a62525aa553e825ee1df8902c 
`/* Cf: L4/L5 RM, Chapter "Independent I/O supply rail" */`

Reply:
Sure, done